avocado/core/data_dir.py

```diff
@@ -30,6 +30,8 @@
 import time
 import tempfile
 
+from six.moves import xrange
+
```

Comment:
Same case. `range` behaves in py3 the same way `xrange` behaves in py2. We should use `range` for them all and, if you find it justifiable, have a `range = xrange` for py2.